test/package.json

```
@@ -1,9 +1,13 @@
 {
   "scripts": {
-    "cypress": "cypress open",
-    "cypress-bs": "browserstack-cypress run",
+    "cypress:local": "cypress open",
```

Need to be explicit about which config file we're using here.
```suggestion
    "cypress:local": "cypress open --config-file ./cypress-local.json",
```